src/vehicle.h

```diff
@@ -1,24 +1,18 @@
-#ifndef _PROTO_GLOBALS_H_
```

Comment:
Lots of duplicate functionality in the vehicle headers, so it made sense to combine them into one.